include/translate-option.php

```diff
@@ -108,11 +110,13 @@ public function translate( $value ) {
 
 		$lang = $GLOBALS['l10n']['pll_string']->get_header( 'Language' );
 
-		if ( ! isset( $this->translated_values[ $lang ] ) ) {
-			$this->translated_values[ $lang ] = $this->translate_string_recursive( $value, $this->keys );
+		$cache = $this->cache->get( $lang );
+		if ( empty( $cache ) ) {
```

Comment:
Shouldn’t we compare to `false` instead?
```suggestion
		if ( false === $cache ) {
```

Reply:
Yes! Fixed in https://github.com/polylang/polylang/pull/1284/commits/b5537ff7b1f34b69010068ab7acd5a794cc3b4d0